composer.json

```diff
@@ -29,8 +29,9 @@
         "symfony/process": "^2.3|^3.0|^4.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.8.35|^5.7",
-        "psr/log": "^1.0"
+        "phpunit/phpunit": "^4.8.35 || ^5.7",
```

Comment:
This is not required anymore

Reply:
it is required if you don't use `simple-phpunit`, which is downloading phar file